test/Release/Gateway/DummyReleaseGateway.php

```diff
@@ -28,7 +28,6 @@ public function __construct()
      */
     public function addRelease( $releaseName, $url )
     {
-        // TODO: Implement addRelease() method.
```

Comment:
This not your code, but if you modify it, "it becomes yours"
so please
return value